JM/CF/Scripts/4_World/CommunityFramework/ModStorage/CF_ModStorageObject.c

```diff
@@ -16,31 +18,44 @@ class CF_ModStorageObject<Class T> : CF_ModStorageBase
 		{
 			mod._ResetStream();
 		}
+
+		m_Module = CF_Modules<CF_ModStorageModule>.Get();
 	}
 
 	override void OnStoreSave(ParamsWriteContext ctx)
 	{
-#ifdef CF_TRACE_ENABLED
+#ifdef CF_MODSTORAGE_TRACE
 		auto trace = CF_Trace_1(this, "OnStoreSave").Add(ctx);
 #endif
 
-		if (GetGame().SaveVersion() < 116)
+		if (GetGame().SaveVersion() < CF_ModStorage.GAME_VERSION_FIRST_INSTALL)
```

Comment:
This was our preparation update we did a while ago? So and data earlier than that we do not attempt to write anything right?
Or what version is `116`? I don't remember the numbering scheme for it.

Reply:
IIRC storage version 116 corresponds to game version 1.10